docs/config/README.md

```diff
@@ -1150,6 +1151,79 @@ trim_at = ".companyname.com"
 disabled = false
 ```
 
+## Helm
+
+The `helm` module shows the currently installed version of Helm.
+The module will be shown if any of the following conditions are met:
+
+- The current directory contains a `helmfile.yaml` file
+- The current directory contains a `Chart.yaml` file
+
+### Options
+
+| Option     | Default                            | Description                                    |
+| ---------- | ---------------------------------- | ---------------------------------------------- |
+| `format`   | `"via [$symbol$version]($style) "` | The format for the module.                     |
+| `symbol`   | `"⎈ "`                            | A format string representing the symbol of Helm. |
+| `style`    | `"bold blue"`                      | The style for the module.                      |
+| `disabled` | `false`                            | Disables the `helm` module.                  |
+
+### Variables
+
+| Variable | Example   | Description                          |
+| -------- | --------- | ------------------------------------ |
+| version  | `v3.1.1` | The version of `helm`                  |
+| symbol   |           | Mirrors the value of option `symbol` |
+| style\*  |           | Mirrors the value of option `style`  |
+
+\*: This variable can only be used as a part of a style string
+
+### Example
+
+```toml
+# ~/.config/starship.toml
+
+[helm]
+format = "via [⎈ $version](bold blue) "
+```
+
+## Hostname
```

Comment:
It looks like you have duplicated the hostname entry here.

Reply:
👍 some copy-paste problems. :) removed.